### PR TITLE
Feature/type conversion functions

### PR DESCRIPTION
**Pull request by pawbu**
The three new functions that we need are:
* `string_from_number`
* `integer_from_string`
* `float_from_string`

🏭

### CODE REVIEW

File: java-vtl-model/src/main/java/no/ssb/vtl/model/VTLNumber.java [hunk 1]
```diff
@@ -30,6 +30,8 @@
 
     @Override
     public abstract T get();
+
+    public abstract String toString();
```
Comment (hadrienk) [inline java-vtl-model/src/main/java/no/ssb/vtl/model/VTLNumber.java]: This conflicts with the toString() Object method. I'd rather have this logic - that is only valid in the case of type conversion - directly into their safeInvoke().

File: java-vtl-documentation/docs/js/directives/vtl-dataset.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ define(['angular'], function (angular) {
             };
 
             function createDataset(text, name) {
-                text = text.replace(/^[\r\n]+|\.|[\r\n]+$/g, ' ').trim(); // Remove tailing line bracks
+                text = text.replace(/^[\r\n]+|[\r\n]+$/g, ' ').trim(); // Remove tailing line breaks
```
Comment (hadrienk) [inline java-vtl-documentation/docs/js/directives/vtl-dataset.js]: Hehe, thanks!

File: README.md [hunk 3]
```diff
@@ -191,13 +191,16 @@ Mathematical|trunc|![done][done]
 Mathematical|power, exp, nroot|![done][done]
 Mathematical|ln, log|![done][done]
 Mathematical|mod|![done][done]
+Mathematical|string_from_number|![done][done]
```
Comment (hadrienk) [inline README.md]: Maybe we should put those in a separate table that is just for the methods that are out of the spec?

File: java-vtl-model/src/main/java/no/ssb/vtl/model/VTLNumber.java (hunk 1 above)
Comment (pawbu) [reply java-vtl-model/src/main/java/no/ssb/vtl/model/VTLNumber.java]: Maybe rename it to `asString()`? Then we avoid the conflict.

Comment (hadrienk) [reply java-vtl-model/src/main/java/no/ssb/vtl/model/VTLNumber.java]: It's better, but it still pulls logic about how to convert this type to another inside the class. We should never use this method except in the VTLFunction implementation. In that sense I don't think it belongs here.

**Pull request merged**